puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/test_kafo_parsers_master_pull_request.yaml

```diff
@@ -0,0 +1,45 @@
+- job:
+    name: test_kafo_parsers_master_pull_request
+    description: |
+      Tests a Kafo Parsers pull request under each Ruby and Puppet configuration.
+    concurrent: true
+    logrotate:
+      daysToKeep: 45
+      numToKeep: -1
+    properties:
+      - github:
+          url: https://github.com/theforeman/kafo_parsers
+    parameters:
+      - pr_parameters
+    scm:
+      - git:
+          url: https://github.com/theforeman/kafo_parsers
+          per-build-tag: true
```

Comment:
Not sure what this does. Does it tag every build? If so, I think we can drop this.

Reply:
Just about every job I've looked at in the past month has this, but I'm not opposed to removing it. We don't seem to use it in any meaningful way.

Reply:
From looking at some console output, it seems to just `git tag` and then continue on its merry way.